WordPressKit/RemoteBlog.swift

```diff
@@ -72,7 +72,7 @@ import NSObject_SafeExpectations
         self.options = RemoteBlogOptionsHelper.mapOptions(fromResponse: json)
         self.planID = json.number(forKeyPath: "plan.product_id")
         self.planTitle = json.string(forKeyPath: "plan.product_name_short")
-        self.hasPaidPlan = json.number(forKeyPath: "plan.is_free")?.boolValue ?? false
+        self.hasPaidPlan = !(json.number(forKeyPath: "plan.is_free")?.boolValue ?? false)
```

Comment:
👋 Hey @guarani, I wanted to run a thought by you on this change:

I understand that this effectively reverses the value for the `plan.is_free` boolean so that it resolves the logic issue defined in the[ parent issue](https://github.com/wordpress-mobile/WordPressKit-iOS/issues/579), e.g. it's not a paid plan if the plan is free.

Is there any concern here about the _default_ value of `false` also being reversed? Before, if a `nil` value was found `hasPaidPlan` would default to `false`, but we're now defaulting to `true`.

Reply:
Good call! I think updating the default value to `true` is the right move.